Android/JavaxInject/License.md

```diff
@@ -0,0 +1,18 @@
+**Xamarin is not responsible for, nor does it grant any licenses to, third-party packages. Some packages may require or install dependencies which are governed by additional licenses.**
```

Comment:
@Redth not sure if this should be Xamarin or Microsoft.

I think is is MS now.

Reply:
I copied GoogleDagger folder then changed text to Javax.Inject wherever required.